src/main/java/uk/gov/pay/api/agreement/model/Agreement.java

```diff
@@ -22,26 +23,42 @@ public class Agreement {
     private String userIdentifier;
 
     @JsonProperty("agreement_id")
+    @Schema(description = "The unique ID GOV.UK Pay automatically associated with this agreement when you created it.", 
+            example = "cgc1ocvh0pt9fqs0ma67r42l58")
     public String getExternalId() {
         return externalId;
     }
 
+    @Schema(description = "The reference you sent when creating this agreement.",
+            example = "CT-22-23-0001")
     public String getReference() {
         return reference;
     }
 
+    @Schema(description = "The description you sent when creating this agreement.",
+            example = "Dorset Council 2022/23 council tax subscription.")
     public String getDescription() {
         return description;
     }
 
+    @Schema(description = "The status of this agreement. " +
+            "`status value` will automatically be set to created because you’ve just created this agreement. " +
```

Comment:
> `status value` will automatically be set to created because you’ve just created this agreement. 

This isn't always true as the `Agreement` model is also used for the response when getting an agreement. I think we could just omit this sentence.